src/common/snippets/src/lowered/linear_ir.cpp

```diff
@@ -44,6 +44,35 @@ LinearIR::LinearIR(const std::shared_ptr<ov::Model>& model, const std::shared_pt
     m_shape_infer = std::make_shared<LIRShapeInfer>(m_expressions, m_io_expressions);
 }
 
+std::shared_ptr<LinearIR> LinearIR::clone() const {
+    auto cloned = std::make_shared<LinearIR>();
+    cloned->m_config = m_config;
+    // Clone underlying ngraph nodes
+    NodeVector original_nodes;
+    original_nodes.reserve(m_expressions.size());
+    for (const auto& expr : m_expressions)
+        original_nodes.push_back(expr->get_node());
+    // node_map and expr_map map original node pointer (expression) to a new pointer (expression)
+    ngraph::NodeMap node_map;
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    ngraph::clone_nodes(original_nodes, node_map);
+    OPENVINO_SUPPRESS_DEPRECATED_END
```

Comment:
It looks like I missed something: why do we need to clone `ov::Node` here if we clone it inside `deep_copy_range` too?

Reply:
No-no, you're right it was there well before a proper deep_copy_range was implemented, but we don't need it anymore. Thanks a lot!